language/tools/disassembler/src/disassembler.rs

```diff
@@ -20,30 +20,36 @@ use move_binary_format::{
 use move_core_types::identifier::IdentStr;
 use move_coverage::coverage_map::{ExecCoverageMap, FunctionCoverage};
 use move_ir_types::location::Loc;
+use move_lang::compiled_unit::{CompiledUnit, NamedCompiledModule, NamedCompiledScript};
+use structopt::StructOpt;
 
 /// Holds the various options that we support while disassembling code.
-#[derive(Debug, Default)]
+#[derive(Debug, Default, StructOpt)]
 pub struct DisassemblerOptions {
     /// Only print non-private functions
+    #[structopt(long = "only-public")]
     pub only_externally_visible: bool,
 
     /// Print the bytecode for the instructions within the function.
+    #[structopt(long = "print-code")]
     pub print_code: bool,
 
     /// Print the basic blocks of the bytecode.
+    #[structopt(long = "print-basic-blocks")]
     pub print_basic_blocks: bool,
 
     /// Print the locals inside each function body.
+    #[structopt(long = "print-locals")]
     pub print_locals: bool,
 }
 
 impl DisassemblerOptions {
     pub fn new() -> Self {
         Self {
             only_externally_visible: false,
-            print_code: false,
```

Comment:
Seems like we are changing default behavior here? What would be its impact?

Reply:
This isn't really used anywhere (only in the CLI). Only change will be that it will print the code out by default -- which is what (IMO) we want default behavior to be.